app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Accordion.kt

```diff
@@ -0,0 +1,145 @@
+package com.hedvig.android.design.system.hedvig
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.design.system.hedvig.AccordionDefaults.Size
+import com.hedvig.android.design.system.hedvig.tokens.AccordionTokens
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.TextAccordion
+import com.hedvig.android.design.system.hedvig.tokens.ColorSchemeKeyTokens.TextPrimary
+
+@Composable
+fun AccordionList(items: List<AccordionData>, modifier: Modifier = Modifier, size: Size = Size.Small) {
+  Column(modifier) {
+    for ((index, perilItem) in items.withIndex()) {
+      var isExpanded by rememberSaveable { mutableStateOf(false) }
+      AccordionItem(
+        isExpanded = isExpanded,
+        onClick = {
+          isExpanded = !isExpanded
+        },
+        title = perilItem.title,
+        description = perilItem.description,
+        size = size,
+      )
+      if (index != items.lastIndex) {
+        Spacer(Modifier.height(4.dp))
+      }
+    }
+  }
+}
+
+@Composable
+fun AccordionItem(
```

Comment:
Do we expect to use this item as an individual composable without it being in a list?
If not we could have this be private I think